docs/fundamentals/orleans.md

```diff
@@ -0,0 +1,179 @@
+---
+title: .NET Aspire with Orleans
+description: Learn how to use Orleans with .NET Aspire
+ms.date: 05/3/2024
+ms.topic: overview
+---
+
+# Use Orleans with .NET Aspire
+
+## Overview
+
+Orleans and .NET Aspire are built for each other. Aspire's application model lets you describe the services, databases, and other resources/infrastructure in your application and how they relate. Orleans' provides a straightforward way to build distributed applications which are elastically scalable and fault-tolerant. Aspire is used to configure and orchestrate Orleans and its dependencies, such as a by providing Orleans with databases cluster membership and storage.
+Orleans is represented as a resource in Aspire. The Orleans resource includes configuration which your service needs to operate, such as cluster membership providers and storage providers.
+
+## Prerequisites
+
+- Orleans version 8.1.0 or later
```

Comment:
We'd likely want to include all the prerequisites, so that if someone comes here from an Orleans point of view, they have the full picture of what's expected. That means, all the setup and tooling of .NET Aspire. We could probably just link to that doc from here.

Reply:
Is there something I can link or include here?

Reply:
Maybe just link to it, with a "For more information, see [setup and tooling]".